### ISSUE TITLE
Auto send emails when customer lands on landing page.

### ISSUE THREAD

**Issue by santhoshb-msft** [Azure/Commercial-Marketplace-SaaS-Accelerator#299]
**Is your feature request related to a problem? Please describe.**
Auto send emails when customer lands on landing page.

**Describe the solution you'd like**
Auto send emails when customer lands on landing page.

**Describe alternatives you've considered**

**Additional context**

**Comment by dstarr** [comment]
Please help me understand why this behavior instead of when the landing page is submitted.

**Comment by santhoshb-msft** [comment]
@dstarr currently unless the customer hits the submit button, there is know way for the publisher to know they have a customer who subscribed to their offer. If the landing page is not so relevant for some publishers and all they want to know if they have a customer subscribed, then this could be valuable. I understand all we are saving is just a button click, but the downside of customer not doing it would be that publisher would not know unless they go to their admin portal. Open to discussion.  

Ideally a publisher should email once they subscribe on the Azure itself. But as we cannot control that, hence this.

**Comment by dstarr** [comment]
Makes sense. I get it now. Unfortunately, the customer is not guaranteed to
actually hit the landing page and they will still have a subscription. This
is why I want a webhook event for a new subscription created. Since we
don't have that, I see the value here.

Got it. Thanks!
